controllers/user-controllers.js

```diff
@@ -0,0 +1,46 @@
+import { User } from "../schema/user.js";
+import jwt from "jsonwebtoken";
+import "dotenv/config";
+import { updateUser } from "../models/users.js";
+
+export const signup = async (req, res) => {
+  const { email, password } = req.body;
+  const userExists = await User.findOne({ email });
+  if (userExists) {
+    return res.status(409).json({ message: "Email in use" });
+  }
+  const newUser = new User({ email, subscription });
+  newUser.setPassword(password);
+  await newUser.save();
+  const savedUser = await User.findOne({ email });
+  return res.status(201).json({ savedUser });
+};
+
+const secret = process.env.SECRET;
```

Comment:
To lepiej na gorze trzymać pliku i nawet zrobić tak
```suggestion
const SECRET_JWT = process.env.SECRET;
```